### ISSUE TITLE
Can this be made to work offline?

### ISSUE THREAD

**Issue by samholmes** [jfromaniello/sublime-html-to-jade#2]
I was on the transit and I wasn't able to use this tool because I had no network connection.

**Comment by jfromaniello** [comment]
Yes we can do something like this... if you have globally installed with npm [html2jade](https://github.com/donpark/html2jade) we can use that instead of the REST api from Aaron.

So, you will have to do beforehand:

```
npm install -g html2jade
```

does this make sense for you?

thanks

**Comment by samholmes** [comment]
I've installed html2jade globally, yet the sublime text plugin still doesn't work offline. Do I need to do something else?

**Comment by jfromaniello** [comment]
it is not implemented yet! was only a suggestion :) I will have a look soon but should be easy to implement

**Comment by samholmes** [comment]
Ah I see. Then, I suggest not requiring users to install a module globally. Maybe there would be a way to include this module as a dependency to the plugin?

**Comment by dflourusso** [comment]
[This](https://github.com/dflourusso/sublime-html2jade) one works offline, but you need to install html2jade too: `npm install -g html2jade`
